frontend/src/components/UserSignUp/EventCard.jsx

```diff
@@ -0,0 +1,82 @@
+import React from "react";
+import PropTypes from "prop-types";
+import styled from "styled-components";
+
+// Displays volunteer event information.
+
+export default function EventCard({ event }) {
+  const EventCardWrapper = styled.div`
+    position: absolute;
+    width: 35%;
+    height: 30%;
+    left: 0%;
+    right: 0%;
+    top: 0%;
+    bottom: 0%;
+    background: #c1d741;
+    border-radius: 30px;
+    padding: 2%;
+  `;
+
+  const Header = styled.h1`
+    display: flex;
+    flex-direction: row;
+    position: static;
+    left: 0%;
+    top: 0%;
+    font-family: Urbanist, default;
+    font-style: normal;
+    font-weight: bold;
+    font-size: 100%;
+    line-height: 43px;
+    color: #003c45;
+    padding: 0%;
+    margin: 0%;
+    border: 0%;
+  `;
+
+  const Name = styled.h1`
+    padding: 0%;
+    margin: 0%;
+    border: 0%;
+    font-size: 36px;
+  `;
+
+  const Slots = styled.h1`
+    padding: 0%;
+    margin: 0%;
+    border: 0%;
+    padding-left: 30%;
+    font-size: 36px;
+  `;
+
+  const Subheader = styled.h2`
+    position: static;
+    left: 0%;
+    right: -110.06%;
+    top: 0%;
+    bottom: 71.4%;
+    font-family: Urbanist, default;
+    font-style: normal;
+    font-weight: normal;
+    font-size: 24px;
+    line-height: 29px;
+    color: #003c45;
+  `;
+
+  return (
+    <EventCardWrapper>
+      <Header>
+        <Name>{event.title}</Name>
+        <Slots>{`${event.slots} slots / 20`}</Slots>
+      </Header>
+      <Subheader>{event.start.toLocaleDateString("en-US")}</Subheader>
+      <Subheader>{event.start.toLocaleTimeString("en-US")}</Subheader>
```

Comment:
I know there was no formatting guidance here, but I think it would look nice if we formatted the date and time a bit more. Justin will get the final say, but I think if the start date was 1/25/2022 9:00:00 and the end date was 1/25/2022 12:00:00, the date should be "Tuesday, January 25" and the time should be "9:00am-12:00pm". I'd highly recommend using [day.js](https://www.npmjs.com/package/dayjs) or [moment](https://www.npmjs.com/package/moment). If you don't have time though, feel free to skip this and I'll add it to another issue in the future.